scripts/qa_crowbarsetup.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 # based on https://github.com/SUSE/cloud/wiki/SUSE-Cloud-Installation-Manual
 
-[[ $(uname -m) =~ ^(aarch64|x86_64)$ ]] || { echo "ERROR: need 64bit" ; exit 1 ; }
```

Comment:
I think, we could even drop this line completely. If someone really finds a i586 VM these days, he will notice a bit later that the repos dont work.